docs/getting-started/quickstart.md

```diff
@@ -74,7 +74,7 @@ The fastest way to get started with Prefect is to add a `@flow` decorator to you
 [Flows](/concepts/flows/) are the core observable, deployable units in Prefect and are the primary entrypoint to orchestrated work.
 
 ```python hl_lines="2 5" title="my_gh_workflow.py"
```

Comment:
Ooo nice, is this an extension that mkdocs adds, or is this Github flavored?

Reply:
highlighting and title are from mkdocs material.